client/components/dashboard/dashboard-service.js

```diff
@@ -79,7 +79,7 @@ const WidgetType = explorer.models.WidgetType;
 explorer.components.dashboard.DashboardService = function(arrayUtilService,
     errorService, widgetFactoryService, dashboardDataService,
     queryBuilderService,  dashboardVersionService, configService,
-    $filter, $location, $rootScope, $timeout, $window) {
+    explorerStateService, $filter, $location, $rootScope, $timeout, $window) {
```

Comment:
The key point here is that user-authored code will never call this constructor.  When the service gets registered, ie: app.Service('myService', MyService) is registered, any parameters on the MyService constructor will be injected as services.